barretenberg/cpp/src/barretenberg/serialize/msgpack_apply.hpp

```diff
@@ -18,14 +18,8 @@ template <msgpack_concepts::HasMsgPack T> void msgpack_apply(const auto& func, a
  */
 template <msgpack_concepts::HasMsgPack T> void msgpack_apply(const T& value, const auto& func)
 {
-    auto static_checker = [&](auto&... value_args) {
```

Comment:
Why did we get rid of this?

Reply:
It would've forced us to add constructors to everything we need to serialise i.e. for something like relation parameters it needed RelationParameter(..all parameters..) so Adam said it's okay to remove it